.gitignore

```diff
@@ -65,3 +65,6 @@ target/
 /hacks_mbof/extras/
 /.vagrant
 *.sqlite3
+
+
+.DS_Store
```

Comment:
I think `.DS_Store` belongs in your `~/.gitignore_global`, not this file.   That's where I have it.  I don't remember putting it there, either, so I think a git tool I use added it for me.

Also, doesn't git automatically ignore those, even if it's not mentioned in a gitignore file?  I thought it did, either because it detects the OS being used or it's just part of the system-wide configuration.